sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -808,6 +808,68 @@ object RapidsConf {
       .booleanConf
       .createWithDefault(false)
 
+  private def getRangeWindowTypeDesc(dt: String, lower: String, upper: String) =
+    s"When order by column is $dt type and the range boundary calculated for a row has overflow, " +
+      s"Spark CPU will get different result comparing GPU.<br>For example, let's assume we have " +
+      s"two columns, one partition column named `uid` and the other column named `dollars`. " +
+      s"And the data is like below,<br>" +
+      s"<code>" +
```

Comment:
I don't think a config description is an appropriate place for an involved example with a sample table.  Check out both the generated .md file and how it looks in Github, it's kind of a mess in both cases.  IMO this should just state the possibility for overflow.  Any detailed discussion like the example here should be put in the windowing section in `docs/compatibility.md` which should be updated as part of this PR.

Reply:
Done.